debezium-ddl-parser/src/test/resources/mysql/examples/ddl_create.sql

```diff
@@ -184,3 +184,32 @@ BEGIN
     RETURN result;
 END;
 #end
+#begin
+-- From MariaDB 10.4.3, the JSON_VALID function is automatically used as a CHECK constraint for the JSON data type alias in order to ensure that a valid json document is inserted.
```

Comment:
here is one of the comment you were looking for. I think they best fit here. WDYT @gunnarmorling ?